<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentsForm</class>
 <widget class="QDialog" name="PaymentsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>514</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Payments</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../app/icons.qrc">
    <normaloff>:/add_payment</normaloff>:/add_payment</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QTableView" name="view">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>831</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="delete_payment_tool_button">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>530</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/delete</normaloff>:/delete</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>901</width>
     <height>73</height>
    </rect>
   </property>
   <property name="title">
    <string>ADD PAYMENT:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>29</y>
      <width>869</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="date_line_edit">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>22</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>ddmmyyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amount">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Rate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rate"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Info:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="info_lineedit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="add_payment_tool_button">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../app/icons.qrc">
         <normaloff>:/new</normaloff>:/new</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="document_line_edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>22</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Document:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>102</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Owing:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="owing_lineedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>22</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>72</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_linedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>22</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>71</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="document_date_line_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>301</width>
        <height>22</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>40</y>
     <width>357</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Partner:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="partner_line_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>301</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="all">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>788</x>
     <y>574</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>date_line_edit</tabstop>
  <tabstop>amount</tabstop>
  <tabstop>rate</tabstop>
  <tabstop>info_lineedit</tabstop>
  <tabstop>add_payment_tool_button</tabstop>
  <tabstop>owing_lineedit</tabstop>
  <tabstop>partner_line_edit</tabstop>
  <tabstop>document_line_edit</tabstop>
  <tabstop>document_date_line_edit</tabstop>
  <tabstop>view</tabstop>
  <tabstop>delete_payment_tool_button</tabstop>
  <tabstop>all</tabstop>
  <tabstop>total_linedit</tabstop>
 </tabstops>
 <resources>
  <include location="../app/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>PaymentsForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>819</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
